projects/packages/status/src/class-host.php

```diff
@@ -142,9 +142,13 @@ public function get_source_query() {
 	 * @return string
 	 */
 	public function get_nameserver_dns_records( $domain ) {
-		$dns_records = dns_get_record( $domain, DNS_NS ); // Fetches the DNS records of type NS (Name Server)
 		$nameservers = array();
 
+		if ( ! function_exists( 'dns_get_record' ) ) {
+			return $nameservers;
```

Comment:
If we are returning early, I think it makes more sense to be explicit and return an empty array directly.
Then maybe move the `$nameservers` declaration after the check for clarity purposes to.

Reply:
Done in 17671f27502dc16b0223e92e029f6b95e27a090c